bom-latest/pom.xml

```
@@ -10,7 +10,7 @@
     <packaging>pom</packaging>
     <properties>
         <configuration-as-code-plugin.version>1.34</configuration-as-code-plugin.version>
```

Test dependencies in the git plugin which test configuration as code support require a newer version of the configuration as code plugin.
```suggestion
        <configuration-as-code-plugin.version>1.35</configuration-as-code-plugin.version>
```

---

Needs a lot more than that to bump this version,
https://github.com/jenkinsci/bom/pull/172

currently failing on cloudbees-folder plugin and I don't understand the error, something to do with a plugin that was split from core but is still picking up the split-plugin instead of the correct version

https://ci.jenkins.io/blue/organizations/jenkins/Tools%2Fbom/detail/PR-172/17/pipeline